src/app/core/services/qbo/qbo-configuration/qbo-connector.service.ts

```
@@ -38,16 +38,17 @@
     cacheBusterObserver: qboCredentialsCache$
   })
   disconnectQBOConnection(): Observable<QBOCredential> {
     globalCacheBusterNotifier.next();
     return this.apiService.patch(`/workspaces/${this.workspaceService.getWorkspaceId()}/credentials/qbo/`, {});
   }
-
-  @Cacheable()
-  getPreferences(): Observable<{}> {
-    return this.apiService.get(`/workspaces/${this.workspaceService.getWorkspaceId()}/qbo/preferences/`, {});
-  }
+  
```

ESLint

[error] Trailing spaces not allowed.

(no-trailing-spaces)